fury/tests/test_utils.py

```diff
@@ -300,3 +304,141 @@ def test_check_order():
                           [2, 3, 4]])
 
     npt.assert_equal(test_tri2, utils.check_order(test_vert, test_tri))
+
+
+def test_vertices_from_actor():
```

Comment:
I suppose this is not part of your PR.

Reply:
These were the changes made by me:
[9534d58](https://github.com/fury-gl/fury/pull/217/commits/9534d587a2d5ea9adba627deb37da103314c595a)
[d16e5c1](https://github.com/fury-gl/fury/pull/217/commits/d16e5c1a19b1a6cae5c97b182097aa36865251f4)
[e65e841](https://github.com/fury-gl/fury/pull/217/commits/e65e84133a80556356e22d8c443245ccdccb394f)

I have rebased and updated the branch as asked.

Reply:
it seems you still have an issue because I still see previous changes in this PR. Furthermore, there are almost 20 commits. It seems a lot for such a change. this commit ea5e8ca shows that you are an author which is not the case.
Please, can you clean your Pull request?

Thank you

Reply:
I am sorry about the mistake. But it seems branch of one more is colliding with mine. Can I make a different PR on a new branch and close this one?

Reply:
Ok, No problem.

Reply:
Thanks! New PR : #218